src/cli/Create.cpp

```diff
@@ -121,6 +121,7 @@ int Create::execute(const QStringList& arguments)
 
     QSharedPointer<Database> db(new Database);
     db->setKey(key);
+    db->setInitialized(true);
```

Comment:
I like the explicit initialisation, but couldn't be determine this automatically somehow based on a set of criteria? It seems rather error-prone to me to rely on the user to call this function.

Reply:
I had the same thought as well

Reply:
Then we should go with that.

Reply:
Done